server/utils/radioMatrixTable.test.ts

```diff
@@ -0,0 +1,96 @@
+import { cell, row, radioMatrixTable } from './radioMatrixTable'
+
+describe('radioMatrixTable', () => {
+  describe('cell', () => {
+    it('returns the markup given the name and preference', () => {
+      expect(cell('name', 'preference')).toMatchStringIgnoringWhitespace(`<td class="govuk-table__cell">
+        <div class="govuk-radios" data-module="govuk-radios">
+            <div class="govuk-radios__item">
+              <input class="govuk-radios__input" id="name-preference" name="name" type="radio" value="preference">
+              <label class="govuk-label govuk-radios__label" for="name-preference"></label>
+            </div>
+          </td>`)
+    })
+
+    it('if passed a check parameter it adds the "checked" tag', () => {
+      expect(cell('name', 'preference', true)).toMatchStringIgnoringWhitespace(`<td class="govuk-table__cell">
+        <div class="govuk-radios" data-module="govuk-radios">
+            <div class="govuk-radios__item">
+              <input class="govuk-radios__input" id="name-preference" name="name" type="radio" value="preference" checked >
+              <label class="govuk-label govuk-radios__label" for="name-preference"></label>
+            </div>
+          </td>`)
+    })
+  })
+
+  describe('row', () => {
```

Comment:
Again, because we're already testing `cell` here, we can probably just test what arguments `cell` gets called with